README.md

```diff
@@ -60,6 +60,29 @@ manually, try the [manual build setup][manual-build].
 
 See also [Windows Troubleshooting Tips][windows-tips].
 
+### Android
+
+- Ensure that the following environment variables are set:
+  - `ANDROID_SDK_ROOT`
+  - `ANDROID_NDK_ROOT`: `$ANDROID_SDK_ROOT/ndk/ANDROID_SDK_ROOT`
```

Comment:
Not sure how to parse this. Is ``$ANDROID_SDK_ROOT/ndk/ANDROID_SDK_ROOT` supposed to the default path? Is the last component here i.e `ANDROID_SDK_ROOT` needed?

Reply:
Oh, this should have read `$ANDROID_SDK_ROOT/ndk/25.2.9519653/`. I've fixed this. Thanks for pointing it out!